### PR TITLE
Add Player To Lobby

### PR DESCRIPTION
**Pull request by ajarrett29**
# Changes Made
Add player to lobby
# Testing Done

### CODE REVIEW

File: src/main/java/com/shepherdjerred/capstone/server/GameServer.java [hunk 1]
```diff
@@ -49,7 +53,11 @@ public void addChatMessage(ChatMessage message) {
   }
 
   public void addPlayer(ClientId clientId, Player player) {
-    handlePlayerMap.put(clientId, player);
+    if (!lobby.isFull()) {
```
Comment (shepherdjerred) [inline src/main/java/com/shepherdjerred/capstone/server/GameServer.java:57]: Throw an exception if the lobby is full

**Review by shepherdjerred — CHANGES_REQUESTED** [review]

**Comment by ajarrett29** [comment]
I am new to java exceptions and I really haven't used exceptions much in general.  Let me know what you think.

**Review by shepherdjerred — APPROVED** [review]

**Comment by shepherdjerred** [comment]
Merge it once your changes in common are made.

**Comment by shepherdjerred** [comment]
Also, the build is failing because there's a test I made that runs infinitely. You might want to disable it haha.

**Review by shepherdjerred — APPROVED** [review]

**Pull request merged**